VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.scss

```diff
@@ -0,0 +1,4 @@
+a {
+  text-decoration: none;
+  padding-left: 0.2em;
+}
```

Comment:
This is duplicated quite a bit. We should probably give them a class and put in a generic css files.